internal/common/mqtt/mqttconnection_test.go

```diff
@@ -0,0 +1,16 @@
+package mqtt
```

Comment:
Need a copyright and license statement at the head of file here.

Reply:
Done!